ste/sender-pageBlob.go

```diff
@@ -53,6 +53,12 @@ type pageBlobSenderBase struct {
 	// Using a automatic pacer here lets us find the right rate for this particular page blob, at which
 	// we won't be trying to move the faster than the Service wants us to.
 	filePacer autopacer
+
+	// destPageRangeOptimizer is necessary for managed disk imports,
+	// as it helps us identify where we actually need to write all zeroes to.
+	// Previously, if a page prefetched all zeroes, we'd ignore it.
+	// This would cause corruption on managed disks.
```

Comment:
This comment "would cause corruption" is pretty scary, and also too vague.  To make it more precise, and therefore also less scary, can you reword it to something like: "this could cause corruption if non-zero values had previously been uploaded to the same page in the same md-impex URL. E.g. by previously uploading a different VHD to the same URL"